mne_microstates.py

```diff
@@ -19,13 +19,14 @@
 from mne.utils import logger, verbose
 
 
-__version__ = '0.4dev0'
+__version__ = '0.5devkb0'
```

Comment:
we are currently working towards version 0.4, so the version string should be `0.4dev0` for now. After your PR is merged, it's time to release version 0.4 and only after that we will change this string to `0.5dev0`.